lib/livebook/hubs.ex

```diff
@@ -112,4 +118,84 @@ defmodule Livebook.Hubs do
   defp to_struct(%{id: "local-" <> _} = fields) do
     Provider.load(%Local{}, fields)
   end
+
+  @doc """
+  Connects to the all available and connectable hubs.
+
+  ## Example
+
+      iex> connect_hubs()
+      :ok
+
+  """
+  @spec connect_hubs() :: :ok
+  def connect_hubs do
+    for hub <- fetch_hubs(), Provider.connectable?(hub) do
+      Provider.connect(hub)
+    end
+
+    :ok
+  end
+
+  @doc """
+  Gets a list of connected hubs.
+
+  ## Example
+
+      iex> fetch_connected_hubs()
+      %{"enterprise-12345" => %{pid: #PID<0.178.0>, hub: %Enterprise{}}}
+
+  """
+  @spec fetch_connected_hubs() :: connected_hubs()
+  def fetch_connected_hubs do
```

Comment:
`get_connected_hubs`?

Reply:
Also, any reason not to return a list? We can build a map whenever needed (though there are going to be at most a couple hubs, so in most cases working with a list is also fine).